records_mover/db/postgres/copy_options/date_input_style.py

```diff
@@ -165,10 +275,33 @@ def upgrade_date_order_style(style: DateOrderStyle, hint_name: str) -> None:
         #
         #  postgres=#
 
+        # Supported!
+        quiet_remove(unhandled_hints, 'timeonlyformat')
+    elif timeonlyformat == "HH:MI:SS":
+        #  postgres=# select time '13:00:00';
+        #     time
+        #  ----------
+        #   13:00:00
+        #  (1 row)
+        #
+        #  postgres=#
+        quiet_remove(unhandled_hints, 'timeonlyformat')
+    elif timeonlyformat == "HH24:MI":
+
+        # "HH24:MI" (e.g., "13:00")
+        #
+        #  postgres=# select time '13:00';
+        #     time
+        #  ----------
+        #   13:00:00
+        #  (1 row)
+        #
+        #  postgres=#
+
         # Supported!
         quiet_remove(unhandled_hints, 'timeonlyformat')
     else:
-        cant_handle_hint(fail_if_cant_handle_hint, 'datetimeformat', hints)
+        cant_handle_hint(fail_if_cant_handle_hint, 'timeonlyformat', hints)
```

Comment:
Whoops, we were printing out the wrong error message here.  Found while debugging a test failure.